src/components/MainPage/MainPage.js

```diff
@@ -28,6 +28,18 @@ function MainPage(props) {
       )
     })
   }
+  let updateUser = async (id) => {
+    let response = await getUser(id)
+    let userInfo = JSON.stringify(response.data.getUser)
+    localStorage.setItem('user', userInfo)
+  }
+
+  useEffect(() => {
+    let user = JSON.parse(localStorage.getItem('user'));
+    if (user) {
+      updateUser(user.id)
+    }
+  })
```

Comment:
Correct me if I am wrong, this is checking if a user exists (i.e logged in) and if they are, update the localStorage with the most up to date information from the DB?

Reply:
Yep!